vault/changelog.release.2021-11-16.md

```diff
@@ -0,0 +1,55 @@
+---
+id: NWuXh9ZreX3XKI6NhRIBJ
+title: "0.69"
+desc: ''
+updated: 1637082185375
+created: 1637081497904
+---
+
+Dendron 0.69 has sprouted  🌱
+
+Updates include enhancements to using Lookup, troubleshooting schema errors, and tweaks to [[Native Workspaces|dendron.topic.workspace.native]].
```

Comment:
Updates include enhancements to  lookup, troubleshooting schema errors, and better support for [[native workspaces|dendron.topic.workspace.native]]